example/file.hpp

```diff
@@ -72,7 +72,14 @@ class file
         boost::json::error_code& ec)
     {
         close();
+#if defined(_MSC_VER)
+# pragma warning( push )
+# pragma warning( disable : 4996 )
+#endif
```

Comment:
for clarity of exposition wouldn't it be better to wrap the whole file in this pragma instead of just the one line?